documentation/book/con-customizing-pod-disruption-budgets.adoc

```diff
@@ -0,0 +1,47 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-customizing-deployments.adoc
+
+[id='con-customizing-pod-disruption-budgets-{context}']
+= Customizing Pod Disruption Budgets
+
+{ProductName} automatically creates a Pod Disruption Budget for every `StatefulSet` or `Deployment` which it creates.
+By default, the Pod Disruption Budgets always allow up to one Pod to be unavailable at a time by setting the `maxUnavailable` value in `PodDisruptionBudget.spec` to `1`.
+You can change this default setting using the Pod Disruption Budget template.
+
+In the Pod Disruption Budget template you can - in addition to Labels and Annotations - customize the `maxUnavailable` field as well:
```

Comment:
This sticks out at me as being a restatement of the previous sentence. I think we could remove this.